workflow/README.md

```diff
@@ -0,0 +1,47 @@
+# TPS Pipeline
```

Comment:
I'm not certain what our final directory structure will be.  Using `workflow` instead of `visualization_scripts` makes sense for now, because we can also put any pre-processing scripts into `workflow`.